cmd/cloud/server.go

```diff
@@ -274,23 +273,25 @@ func executeServerCmd(flags serverFlags) error {
 
 	kopsProvisioner := provisioner.NewKopsProvisioner(
 		provisioningParams,
-		resourceUtil,
-		logger,
 		sqlStore,
-		provisioner.NewBackupOperator(flags.backupRestoreToolImage, awsConfig.Region, flags.backupJobTTL),
+		logger,
 	)
-	defer kopsProvisioner.Teardown()
 
-	eksProvisioner := provisioner.NewEKSProvisioner(sqlStore,
-		sqlStore,
+	eksProvisioner := provisioner.NewEKSProvisioner(
 		provisioningParams,
-		resourceUtil,
 		awsClient,
+		sqlStore,
 		logger,
 	)
 
-	provisionerOptionForSupervisor := supervisor.GetProvisionerOption(eksProvisioner, kopsProvisioner)
-	provisionerOptionForAPI := api.GetProvisionerOption(eksProvisioner, kopsProvisioner)
+	provisionerInterface := provisioner.NewProvisioner(
```

Comment:
Pretty sure this is not an interface, just a provisioner.

Reply:
Right. Thanks. I will rename it